### DISCUSSION TITLE
Turn fork into standalone repo

### DISCUSSION THREAD

**Discussion by NunoDuarte** [community/community#28993] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I'm aware that commits in a fork do not count to my contributions until they are accepted as a pull request. Regardless, I prefer to work on this repo as a standalone. I've sent an email to github support but they direct me here which does not solve my problem. I've checked that there was another person with the same problem and they resolved it by contacting the github support. But that just redirects me here in a vicious cycle. 

Please help. 

Nuno

**Comment by airtower-luna** [comment, marked as answer]
That seems odd! There's a [specific category for forks](https://support.github.com/request/fork) on the support page, did you use the "detach" button there, or how did you send the mail? The button opens a "virtual assistant" by default. I haven't needed to detach a fork to I can't say how well it works, though. :sweat_smile:

**Reply by NunoDuarte** [reply]
Thanks a lot! That is exactly what I was looking for. From the main support page I could not find that section. This solved my issue instantly.

**Comment by delosemre** [comment]
[Have you applied to Github day?](https://support.github.com)

**Reply by NunoDuarte** [reply]
Hi, I'm not familiar with that. The previous answer was enough to solve my issue, though. Thank you anyways.

**Comment by jfullstackdev** [comment]
after making sure you are allowed to have that standalone repo, you can simply create a new repo for that project

**Reply by NunoDuarte** [reply]
Hi, the standalone repo is just a normal repo in my understanding. Creating a new is not necessary. I just wanted to continue working on it and keep track of my commits.

**Reply by radekg** [reply]
Clone the repo locally, delete the fork, create the unforked repo with the same name, push all branches. History is kept.

**Reply by kytpbs** [reply]
This is half wrong, yes git history is kept but PRs, Action secrets/variables, and other GitHub-specific details are removed. please use the other answers
